### PR TITLE
docs: fix filename-access example

### PR DESCRIPTION
**Pull request by yukinakanaka**
Fixes: https://github.com/cilium/tetragon/issues/2334

A demo pod called `xwing` doesn't seem to have `vi` now. This may be because of the image update: https://github.com/cilium/tetragon/pull/2311, https://github.com/cilium/cilium/pull/31373.

So, I propose creating a new `busybox` pod instead of `xwing`.

### CODE REVIEW

**Comment by yukinakanaka** [comment]
You can see the preview here: https://deploy-preview-2335--tetragon.netlify.app/docs/use-cases/filename-access/#examples

File: docs/content/en/docs/use-cases/filename-access.md
Comment (mtardy) [inline docs/content/en/docs/use-cases/filename-access.md:85]: Not related to what you fixed but I think it would be better to use the `tetra` CLI from the container here, so that you don't need to install tetra locally and the version is on par.
```suggestion
kubectl exec -ti -n kube-system ds/tetragon -c tetragon -- tetra getevents -o compact --namespace default --pod file-access
```

File: docs/content/en/docs/use-cases/filename-access.md
Comment (mtardy) [inline docs/content/en/docs/use-cases/filename-access.md:227]:
```suggestion
To delete the `file-access` Pod run, from the interactive bash session, type:
```

File: docs/content/en/docs/use-cases/filename-access.md
Comment (mtardy) [inline docs/content/en/docs/use-cases/filename-access.md:82]:
```suggestion
Next, you can start monitoring the events from a `file-access` Pod:
```

File: docs/content/en/docs/use-cases/filename-access.md
Comment (mtardy) [inline docs/content/en/docs/use-cases/filename-access.md:88]:
```suggestion
In another terminal, we deploy the `file-access` Pod with an interactive shell:
```

File: docs/content/en/docs/use-cases/filename-access.md
Comment (mtardy) [inline docs/content/en/docs/use-cases/filename-access.md:110]:
```suggestion
🚀 process default/file-access /bin/sh
🚀 process default/file-access /bin/vi /etc/passwd
📚 read    default/file-access /bin/vi /etc/passwd
📚 read    default/file-access /bin/vi /etc/passwd
📚 read    default/file-access /bin/vi /etc/passwd
📝 write   default/file-access /bin/vi /etc/passwd
📝 truncate default/file-access /bin/vi /etc/passwd
💥 exit    default/file-access /bin/vi /etc/passwd 0
```

File: docs/content/en/docs/use-cases/filename-access.md
Comment (mtardy) [inline docs/content/en/docs/use-cases/filename-access.md:135]:
```suggestion
        "name": "file-access",
```

File: docs/content/en/docs/use-cases/filename-access.md
Comment (mtardy) [inline docs/content/en/docs/use-cases/filename-access.md:138]:
```suggestion
          "name": "file-access",
```

File: docs/content/en/docs/use-cases/filename-access.md
Comment (mtardy) [inline docs/content/en/docs/use-cases/filename-access.md:149]:
```suggestion
        "workload": "file-access",
```

File: docs/content/en/docs/use-cases/filename-access.md
Comment (mtardy) [inline docs/content/en/docs/use-cases/filename-access.md:171]:
```suggestion
          "name": "file-access",
```

File: docs/content/en/docs/use-cases/filename-access.md
Comment (mtardy) [inline docs/content/en/docs/use-cases/filename-access.md:180]:
```suggestion
          "run": "file-access"
```

**Review by mtardy — COMMENTED** [review]
Thanks a lot, that works fine, I don't mind not using the star wars deployments for this example! I just added a nit to change `file-access-pod` to `file-access` just to avoid the "file-access-pod Pod" stutter!

Another thing is that we could put the `tetra getevents` after the creation of the pod so that people are not confused with all the events that will appear at the pod creation. We can still put it before if we really want it but we should make a note on what are all those events.

This is mostly nits, and correct things the tutorial was not respecting even before (saying Pod instead of pod), so we can merge this anyway if you don't feel like the fix are worth it!

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Good idea! I’ve accepted your suggestion.

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

File: docs/content/en/docs/use-cases/filename-access.md
Comment (yukinakanaka) [reply docs/content/en/docs/use-cases/filename-access.md]: Fixed 👍

**Comment by yukinakanaka** [comment]
Thank you for your kind review! I fixed up my commit by following your suggestion.

- Change the pod name to `file-access` from `file-access-pod`.
- Use `Pod` instead of `pod`.

>Another thing is that we could put the tetra getevents after the creation of the pod so that people are not confused with all the events that will appear at the pod creation. 

That's true. I put tetra getevents after the creation of pod.

**Review by mtardy — APPROVED** [review]
Thanks a lot!!

**Pull request merged**
